internal/ethapi/api.go

```diff
@@ -689,7 +689,13 @@ func (s *BlockChainAPI) GetProof(ctx context.Context, address common.Address, st
 		return nil, err
 	}
 	codeHash := statedb.GetCodeHash(address)
+	if codeHash == (common.Hash{}) {
+		codeHash = types.EmptyCodeHash
+	}
```

Comment:
This fixes this specific API call, but shouldn't this issue be fixed in statedb? Maybe it has some deeper implications?

Reply:
thanks for the advice, now fix it in package statedb